smsgateway/tasks.py

```diff
@@ -94,6 +94,11 @@ def process_smses(smsk, sms_id, account_slug):
 
 
 def recv_smses(account_slug='redistore', async=False):
+    # to sleep more comfortable, we explicitly don't want to receive smses while in WHITELIST mode
+    # (because of shared ESME instances)
+    if WHITELIST:
```

Comment:
It may be better to define independent setting i.e. CONSUME_INCOMING_SMSES for that case and set it explicitly. It will be harder to accidentally turn it on.